lib/ohai/plugins/c.rb

```diff
@@ -39,6 +40,7 @@
         end
       end
     rescue Errno::ENOENT
```

Comment:
Example from a command a little later on in this PR:

```
[2] pry(main)> so = Mixlib::ShellOut.new('elixir -v').run_command
Errno::ENOENT: No such file or directory - elixir
from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/mixlib-shellout-2.2.5/lib/mixlib/shellout/unix.rb:338:in `exec'
```

The only difference is that I have C installed through XCode but Elixir has never been installed on my system?